concordia/static/scss/base.scss

```diff
@@ -811,9 +811,10 @@ body.view-transcriptions--asset-detail main {
 
 .help-center-card {
     color: #fff;
-    width: 300px;
+    // width: 300px;
```

Comment:
If this is not needed it should be removed. If there was any need to get it back in the future, Git has a full history.

Reply:
ok removed the line, and pushed it back up.